.github/workflows/notify-on-pull-requests.yml

```diff
@@ -1,11 +1,24 @@
-name: Notify matrix room when a pull request is merged (or rebased)
+name: Notify matrix room when a pull request is opened (or reopened) or merged (or rebased)
 
 on:
   pull_request:
-    types:
-      - closed
+    types: [opened, reopened, closed]
 
 jobs:
+  ping_matrix_when_opened:
+    if: github.event.pull_request.merged == false
```

Comment:
There seems to be a [bunch of events](github.event.pull_request), and i suspect that this one will be triggered when PR is closed without merging, too. I'm not sure until we run it. Can be fixed later, i suppose

Reply:
Good point!